src/modules/vendor/components/courses/ProfileFollowUp.vue

```diff
@@ -85,14 +119,49 @@ export default {
       smsModal: false,
       messageType: 'convocation',
       messageTypeOptions: [
-        { label: 'SMS de convocation', value: 'convocation' },
-        { label: 'SMS de rappel', value: 'reminder' },
+        { label: 'SMS de convocation', value: CONVOCATION },
+        { label: 'SMS de rappel', value: REMINDER },
+      ],
+      messateTypes: [
+        { label: 'Convocation', value: CONVOCATION },
+        { label: 'Rappel', value: REMINDER },
       ],
       message: '',
       loading: false,
       courseLoading: false,
+      smsSent: [],
+      smsSentColumns: [
+        {
+          name: 'type',
+          label: 'Type',
+          align: 'left',
+          field: 'type',
+          format: (value) => this.getType(value),
+        },
+        {
+          name: 'date',
+          label: 'Date d\'envoi',
+          align: 'left',
+          field: 'date',
+          format: (value) => this.$moment(value).format('DD/MM/YYYY'),
+        },
+        {
+          name: 'actions',
+          label: '',
+          align: 'center',
+          field: '_id',
+        },
+      ],
+      pagination: { rowsPerPage: 0 },
+      smsLoading: false,
+      smsHistoriesModal: false,
+      smsHistory: {},
     };
   },
+  async mounted () {
+    await this.refreshCourse();
+    await this.refreshSms();
+  },
```

Comment:
dans un `created`
je pense qu'on peut commencer a changer sur les composants sur lesquels on passe...
https://stackoverflow.com/questions/45813347/difference-between-the-created-and-mounted-events-in-vue-js

Reply:
Done !